src/services/__tests__/messagesService.test.ts

```diff
@@ -203,6 +206,21 @@ describe("MessageService#getMessagesByUser", () => {
     expect(res.kind).toEqual("IResponseErrorNotFound");
   });
 
+  it("fails to get message by user returns an 429 HTTP error from the upstream API", async () => {
+    mockGetMessages.mockImplementation(() =>
+      t.success(tooManyReqApiMessagesResponse)
+    );
+
+    const service = new MessageService(api);
+
+    const res = await service.getMessagesByUser(mockedUser);
+
+    expect(mockGetMessages).toHaveBeenCalledWith({
```

Comment:
this can be removed, we're just testing that the response is an `IResponseErrorTooManyRequests`

Reply:
Fix in e0e50dd